probe/docker/capabilities.go

```diff
@@ -0,0 +1,20 @@
+package docker
+
+import (
+	"log"
+
+	"github.com/weaveworks/scope/probe/capabilities"
+	"github.com/weaveworks/scope/xfer"
+)
+
+// RestartContainer is the capability ID used by the docker intergation.
+const RestartContainer = "docker_restart_container"
+
+func init() {
+	capabilities.Register(RestartContainer, func(req xfer.Request) xfer.Response {
```

Comment:
In general it's probably better form to pull the func out, so we can unit-test it. Is there such a thing as a capability that will need initialization at (or after) probe boot time?